scripts/gen_lib.lua

```diff
@@ -0,0 +1,71 @@
+lpeg = require('lpeg')
```

Comment:
I'm talking about sensible defaults; how these commands normally are invoked, it doesn't prohibit anyone in an edge case to construct​ the cmdline themselves (_possibly_ by modifying one from verbose `make gen_xx`, if from top-level or build/ is always up to the individual developer and completely besides the point)